packages/repmgr13/repmgr.spec

```
@@ -41,9 +45,19 @@ Requires(postun):   initscripts
 Group:        Applications/Databases
 BuildRoot:        %{_tmppath}/%{name}-%{version}%{extra_version}-%{release}-root-%(%{__id_u} -n)
 %endif
-BuildRequires:    postgresql, postgresql-devel, postgresql-static
-BuildRequires:    libxslt-devel, pam-devel, openssl-devel, readline-devel
-Requires:         postgresql-server
+BuildRequires:    postgresql, postgresql-devel
```

```suggestion
BuildRequires:    postgresql, postgresql-devel, postgresql-static
```

Without this it fails to build with:
```
Building against PostgreSQL 
Makefile.global:26: /usr/lib64/pgsql/pgxs/src/makefiles/pgxs.mk: No such file or directory
make: *** No rule to make target '/usr/lib64/pgsql/pgxs/src/makefiles/pgxs.mk'.  Stop.
error: Bad exit status from /var/tmp/rpm-tmp.QEa250 (%build)
    Bad exit status from /var/tmp/rpm-tmp.QEa250 (%build)
```

---

Good call, I changed this for ubi9 but didn't bring it back here.

---

I'll squash these two spec changes into the "Changes for repmgr13" commit as I wanted to make it clear what was changed from upstream for 13.

---

Ok, done and forced push.